larq_compute_engine/core/bgemm_kernels_common.h

```diff
@@ -130,9 +127,6 @@ inline void MakeBinaryKernelParams(
   params->post_activation_multiplier =
       spec.output_transform.post_activation_multiplier;
   params->post_activation_bias = spec.output_transform.post_activation_bias;
-  if (params->post_activation_multiplier && params->post_activation_bias) {
-    flags |= RUY_ASM_FLAG_HAS_BIAS;
-  }
   params->backtransform_add = spec.output_transform.backtransform_add;
   params->flags = flags;
```

Comment:
Should we remove the `flags` param altogether (this was the only flag being used)?

Reply:
Good point, removed them in a0ca403b78562fcd6c5c5e34324011bdb5fcafbc

I guess if we need them at some point we can add them back rather easily.